startup.py

```diff
@@ -21,6 +21,6 @@
     elif choice == '2':
         dbmanager.stop()
     elif choice == '3':
-        initdb.main("","")
+        initdb.main()
```

Comment:
Mostly poor git practices. The problem here is that you have pulled from another feature-branch. More specially, a feature-branch that is under review still and is still being worked on. That means you are now working off of code that is liable to not be fully working and can be changed at any commit. That is why you need to wait until a feature-branch as been fully reviewed, tested and merged into master before using anything from that feature. And in that case, you still don't merge with that feature branch. You merge with master.